Feature/develop/task2

Bozhyk-Oleh opened the pull request:
lab2

Bozhyk-Oleh committed:
VERSION 1 - wpf client app that turns on and off bult in led of the arduino

Bozhyk-Oleh committed:
VERSION 1 - server app  turns on and off bult in led of the arduino

Bozhyk-Oleh committed:
new version of client app, bridge and server Sketches

Bozhyk-Oleh committed:
relplaced

Bozhyk-Oleh committed:
Delete Client/obj directory

Bozhyk-Oleh committed:
Delete Client.csproj

Bozhyk-Oleh committed:
Delete Client/bin/Debug directory

Bozhyk-Oleh committed:
Delete Client/Properties directory

Bozhyk-Oleh committed:
Delete App.config

Bozhyk-Oleh committed:
removed mistakes))

Bozhyk-Oleh committed:
modified watchdog

Bozhyk-Oleh committed:
Update README.md

task2

Bozhyk-Oleh committed:
Merge branch 'develop' into feature/develop/task2

andrii-s-shpitser-lpnu-ua commented on Client/Client.sln:
```diff
@@ -0,0 +1,25 @@
+﻿
```
please add CI scripts to build your project

andrii-s-shpitser-lpnu-ua commented on master_server/master_server.ino:
```diff
@@ -0,0 +1,67 @@
+  wdt_enable (WDTO_2S);
+}
+
+void loop() {
```
loop is too big, please split it into smaller functions.

Bozhyk-Oleh committed:
master_server: made void loop shorter and removed delays;
slave_bridge:made the program more reliable(max message 12c transfer butter 30 bytes) and removed delays
client: added autoscroll and error message if the input message is bigger than 30 bytes

Bozhyk-Oleh committed:
try ci

Bozhyk-Oleh committed:
try to trigger ci

Bozhyk-Oleh committed:
try to tigger ci

Bozhyk-Oleh committed:
add ci to brach task2

Bozhyk-Oleh committed:
try to build project in ci

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
downgrade to .net 3.0

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update Client.csproj

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

andrii-s-shpitser-lpnu-ua commented on .github/workflows/build_task2.yml:
```diff
@@ -5,7 +5,7 @@ on: [workflow_dispatch]
   build:
 
-    runs-on: microsoft-latest
+    runs-on: ubuntu-latest
```
will it generate EXE for Windows

Bozhyk-Oleh replied:
-- no  i get  an error when trying to build the project  in ci 
  C:\Program Files\dotnet\sdk\6.0.402\Microsoft.Common.CurrentVersion.targets(1220,5): error MSB3644: The reference assemblies for .NETFramework,Version=v3.0 were not found. To resolve this, install the Developer Pack (SDK/Targeting Pack) for this framework version or retarget your application. You can download .NET Framework Developer Packs at https://aka.ms/msbuild/developerpacks

Bozhyk-Oleh replied:
i did try something new and now it is generating a exe file, please check it 
https://github.com/Bozhyk-Oleh/csdt2223ki46bozhyk02/actions/runs/3272637944/jobs/5383887733

andrii-s-shpitser-lpnu-ua replied:
Good, but how I can download EXE?

Bozhyk-Oleh replied:
DONE!
https://github.com/Bozhyk-Oleh/csdt2223ki46bozhyk02/actions/runs/3276904833

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update Client.csproj

Bozhyk-Oleh committed:
change csproj

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
final try

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update build_task2.yml

Bozhyk-Oleh committed:
Update README.md

Bozhyk-Oleh committed:
ci for uploading exe

Bozhyk-Oleh committed:
Merge branch 'develop' into feature/develop/task2
